[website] New wave of open roles (#35240)
mui/material-ui: docs/src/pages/careers/lead-designer.md

# Lead Designer

<p class="description">We are looking for a lead designer to own most of our design work and manage a small design team.</p>

## Details of the Role

- Location: Remote (preference for UTC-6 to UTC+5).
- Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102))

> - Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).

Author: The link is for Engineers. We need to create one for designers too (unless we already have one).

Reviewer: We have one for designers, let's fix the link.

# Lead Designer

> <p class="description">You will own most of MUI's design work and manage a small team of designers.</p>

Reviewer 2: New description: Oversee the current design team of two and manage the design work across Core, X, Toolpad, and Design Kits teams.

Reviewer: I think that the current description place what matters most first: the primary goal is to own most of MUI's design work. Overseeing a team is one aspect of reaching this end. Maybe?

```suggestion
<p class="description">You will be accountable for MUI's design work and manage a small team of designers.</p>
```

We envision a future where MUI becomes the default toolkit for web developers to create UIs.
It's why we started MUI Base, Joy UI, and MUI Toolpad. Design is key to achieving this goal.

## About the role

### What you'll do on a day-to-day basis

Depending on the day, you'll:

- Make the product experience more delightful, e.g. visually more appealing
- Pay attention to all the smallest interface interactions so they yield an outstanding UX
- Develop the brand
- Refresh and expand the marketing pages, e.g. with a showcase & case study section
- Design the next features across the different product
- Shape the direction of the design kit products
- **Solve organization issues**
  - You will improve how the design responsibilities are covered.
  - You will ensure the consistency of design and product direction throughout all the products and all customer-facing artifacts.
- **Exercise team leadership**

>   - You will overseas the overall design quality

Reviewer 2: - You will oversee the overall design quality

  - You will manage designer's career

>   - You will hire and maintain a great team

Reviewer 2: - Ensure high quality UI and UX across teams, libraries, components, and product strategy
- Collaborate with product management and engineering to ensure that design work is aligned with business goals and user needs
- Create design concepts, prototypes, and high-fidelity mockups for projects on the X, Core, and Toolpad teams
- Update and implement new components on the Core, X, and Toolpad teams
- Develop and maintain MUI’s design system and style guide

### As a team leader, you will:

- Manage the design process, including user research, user flows, and user testing
- Review design work and provide feedback to ensure overall design quality and consistency, with consideration to both UI and UX.
- Set goals, priorities, and distribute tasks for the design team
- Provide mentorship and support to two existing design team members
- Hire and retain a great team
- Oversee the Design Kits, ensuring they’re aligned with engineering implementation.

## About you

We're looking for someone with an expert eye for visual and UX design. You have enough development skills (CSS-in-JS, JSX) to implement the designed interfaces. Details matter to you.

### Skills you should have

- **Expert-level knowledge of visual design**. You can create great-looking UIs. You spot small kerning issues without even paying attention which makes you care about the details more than most people.
- **Expert-level knowledge of UX design**. You can build intuitive UIs, both at the scope of an application and of a component micro-interaction.
- **Enough development skills to implement the designed interfaces**. Our React engineers are often busy implementing complex components. We are looking for somebody with enough CSS-in-JS and JSX skills to not depend on them to implement the design.
- **Strong written and verbal communication skills**. As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.

> 

Reviewer 2: - **Management of a design team**. Whether formally or informally, you have shown that you are a design leader and can provide guidance, support, and mentorship to other designers.

### What would be nice if you had, but isn't required

- **You've contributed to a design system before**. You have worked on a library of components in the past, maybe for internal usage in your company.

> - **You have other complementary skills**. We are a tiny team of 6 people. We could use any hidden skills that you might have. Maybe you are great at creating animations or creating illustrations.

Reviewer 2: 2 options — 
Small edit: update the number of our tiny team 
Larger question: Perhaps cut this line? I don't know if we are gaining much by explicitly adding a catch-all for detailed skills when we are more looking for managerial duties.

Reviewer: Ah right, we are not 6 people anymore.
